### PR TITLE
Laravel Fixer

### PR DESCRIPTION
**Pull request by kberzinch**
This pull request includes changes and recommendations for crafting your Laravel application. These are based on the Laravel documentation and widely adopted practices within the community.

**Before merging**, you need to:

- Checkout the `shift-37447` branch
- Review **all** comments for additional changes
- Thoroughly test your application

Don't hesitate to send your feedback to shift@laravelshift.com or share on [Twitter](https://twitter.com/laravelshift).

### CODE REVIEW

**Comment by kberzinch** [comment]
:warning: The following controllers contain actions outside of the 7 resource actions (`index`, `create`, `store`, `show`, `edit`, `update`, `destroy`). For more details, review [the docs](https://laravel.com/docs/5.7/controllers#resource-controllers) or watch [Cruddy by Design](https://www.youtube.com/watch?v=MF0jFKvS4SI) to see if you can rework these into *resource controllers*.

- [ ] app/Http/Controllers/AttendanceController.php
- [ ] app/Http/Controllers/AuthController.php
- [ ] app/Http/Controllers/DuesPackageController.php
- [ ] app/Http/Controllers/DuesTransactionController.php
- [ ] app/Http/Controllers/GitHubController.php
- [ ] app/Http/Controllers/GoogleController.php
- [ ] app/Http/Controllers/NotificationController.php
- [ ] app/Http/Controllers/PaymentController.php
- [ ] app/Http/Controllers/RecruitingCampaignController.php
- [ ] app/Http/Controllers/RecruitingVisitController.php
- [ ] app/Http/Controllers/RedirectController.php
- [ ] app/Http/Controllers/ResumeController.php
- [ ] app/Http/Controllers/RoleController.php
- [ ] app/Http/Controllers/RsvpController.php
- [ ] app/Http/Controllers/TeamController.php
- [ ] app/Http/Controllers/UserController.php

**Comment by kberzinch** [comment]
:warning: Shift detected instances of controller validation still remaining. Shift may have been unable to automate their conversion due to the structure of the code.

You should review these instances for dynamic validation rules or custom exception handling . If you can inline these, feel free to request a rerun of this Shift.

- [ ] app/Http/Controllers/PaymentController.php
- [ ] app/Http/Controllers/RoleController.php

**Comment by kberzinch** [comment]
:information_source: Shift detected your application only has the default tests. You can quickly start adding tests by using the [Test Generator](https://laravelshift.com/laravel-test-generator) to automatically create the model factories, test classes, and sample tests cases for your Laravel application.

**Comment by kberzinch** [comment]
:x: PHP syntax errors were detected after running your Shift. Often these are simply differences between the PHP version of Shift and your project. Occasionally they are misplaced lines.

You can check the PHP syntax by running `php -l` on the following files:

- [ ] app/Http/Controllers/GitHubController.php
- [ ] app/Http/Controllers/GoogleController.php
- [ ] app/Http/Controllers/PaymentController.php

File: app/Models/AttendanceExport.php [hunk 1]
```diff
@@ -67,14 +67,11 @@ class AttendanceExport extends Model
      *
      * @var array<string>
      */
-    protected $dates = [
-        'created_at',
-        'updated_at',
-        'deleted_at',
```
Comment (JoshOldenburg) [inline app/Models/AttendanceExport.php:73]: Are these implied now?

**Review by JoshOldenburg — DISMISSED** [review]
Looks good to me

File: app/Models/AttendanceExport.php (hunk 1 above)
Comment (kberzinch) [reply app/Models/AttendanceExport.php:73]: [Yes](https://laravel.com/docs/8.x/eloquent-mutators#date-casting).

**Pull request merged**